src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -173,6 +173,21 @@ public Category GetCategory
         }
 
         /// <summary>
+        /// Get Element Type
+        /// </summary>
+        public Element ElementType
+        {
+            get
+            {
+                // Get the active Document
+                Autodesk.Revit.DB.Document document = DocumentManager.Instance.CurrentDBDocument;
+
+                return document.GetElement(this.InternalElement.GetTypeId()).ToDSType(true);
```

Comment:
Also, we should update the comments on this property to explain that it might return null (if that's what we decide to return). Otherwise, this will return null and the user won't have any way of knowing why.